### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by almaslennikov**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-    cd /usr/src/k8s-rdma-shared-dp && \
-    make clean && \
+RUN apk add --no-cache --virtual build-base=0.5-r3 linux-headers=5.19.5-r0
+WORKDIR /usr/src/k8s-rdma-shared-dp
+RUN make clean && \
     make build
 
-FROM ${BASE_IMAGE}
-RUN apk add kmod hwdata-pci
+FROM alpine:3.15.4
```
Comment (e0ne) [inline Dockerfile:13]: let's bumb to the latestr image

File: Dockerfile.ubi [hunk 2]
```diff
@@ -1,18 +1,16 @@
...
-RUN apk add --update --virtual build-dependencies build-base linux-headers git && \
-    cd /usr/src/k8s-rdma-shared-dp && \
-    make clean && \
+RUN apk add --update --virtual build-dependencies build-base linux-headers git
+WORKDIR /usr/src/k8s-rdma-shared-dp
+RUN make clean && \
     make build
 
-FROM ${BASE_IMAGE}
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
```
Comment (e0ne) [inline Dockerfile.ubi:13]: let's bumb to the latestr image

File: scripts/deploy.sh [hunk 3]
```diff
@@ -3,8 +3,8 @@ IMAGE_NAME=$1
 VERSION=$2
 CPU_ARCH=$3
 export DOCKER_CLI_EXPERIMENTAL="enabled";
-docker tag $IMAGE_NAME ${IMAGE_NAME}-${CPU_ARCH}:${VERSION}
-docker push ${IMAGE_NAME}-${CPU_ARCH}:${VERSION}
-docker manifest create ${IMAGE_NAME}:${VERSION} ${IMAGE_NAME}-${CPU_ARCH}:${VERSION}
-docker manifest annotate ${IMAGE_NAME}:${VERSION} ${IMAGE_NAME}-${CPU_ARCH}:${VERSION} --arch ${CPU_ARCH}
-docker manifest push ${IMAGE_NAME}:${VERSION}
+docker tag "$IMAGE_NAME" "${IMAGE_NAME}"-"${CPU_ARCH}":"${VERSION}"
```
Comment (e0ne) [inline scripts/deploy.sh:6]: I'm wondering if we still use this script or it could be deleted

**Review by e0ne — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml [hunk 4]
```diff
@@ -34,15 +34,15 @@ jobs:
...
       - name: Set up Go 1.x
         uses: actions/setup-go@v3
         with:
-          go-version: '1.18.x'
+          go-version: '1.20.x'
       - name: checkout
         uses: actions/checkout@v3
       - name: build
-        run: make build
+        run: make build
```
Comment (ykulazhenkov) [inline .github/workflows/build.yml:48]: no new line at the end

**Comment by ykulazhenkov** [comment]
Do we want to support multiarch image build for this repo?

File: scripts/deploy.sh (hunk 3 above)
Comment (almaslennikov) [reply scripts/deploy.sh]: Checked the logs from CI - the script seems to be unused. Deleted it together with the shellcheck scan

**Review by e0ne — APPROVED** [review]
Thanks for addressing my comments

**Review by ykulazhenkov — APPROVED** [review]

**Review by adrianchiris — APPROVED** [review]

**Pull request merged**